misc/metrics/src/swarm.rs

```diff
@@ -372,3 +383,8 @@ impl<TTransErr> From<&libp2p_swarm::PendingInboundConnectionError<TTransErr>>
         }
     }
 }
+
+fn create_connection_duration_histogram() -> Histogram {
+    // Histogram::new((1..99).map(|i|10. * i as f64))
+    Histogram::new(metrics::histogram::exponential_buckets(1.,2.,10))
+}
```

Comment:
Please fix the newline.

Reply:
I'm not 100% certain what you're referring to here. 
The file now ends in a blank line, is that what you're referring to?
I also removed the commented dead code.